src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  tasks: Todo[]
```

Comment:
Your component name is `TodoList` (and everywhere todo), so no need to add another entity which in future will confound others.
```suggestion
  todos: Todo[]
```